### ISSUE TITLE
No support for i9 7900x Temps

### ISSUE THREAD

**Issue by ProtostarMusic** [openhardwaremonitor/openhardwaremonitor#1071]
Only core loads show, no temps/voltages etc.
Shows fine in HWMonitor etc!

https://i.imgur.com/ozJVAHS.png

**Comment by DarthDestroyer** [comment]
Without a Report, there is only one Option :-P

Do you use the latest Version?
http://openhardwaremonitor.org/files/openhardwaremonitor-v0.8.0.3-alpha.zip

**Comment by ProtostarMusic** [comment]
Yup!
https://i.imgur.com/S2qX7Jh.png

Here's a report: https://pastebin.com/LTEW0e8H

**Comment by ProtostarMusic** [comment]
@DarthDestroyer any help on the matter? :)

**Comment by DarthDestroyer** [comment]
I can only lead you to #1005, it should help you with the CPU Data

**Comment by ProtostarMusic** [comment]
@DarthDestroyer  yeah this should help, i have no idea how to utilise this though sadly haha, i dont touch much to do with code, any tuts/advice? :D

**Comment by DarthDestroyer** [comment]
in #981 is it a little bit explained, its not really complicated.

if you can write a word document, you would handle it ;-)

**Comment by ProtostarMusic** [comment]
Any specific packs i need when installing VS? :D https://i.imgur.com/AOE58v8.png

**Comment by DarthDestroyer** [comment]
I cant really read it because of the Resolution, but i think that "top right" is enough!

Should be .net or similar, if i remember correct

**Comment by ProtostarMusic** [comment]
@DarthDestroyer  you can click on the image on imgur to enlarge it!

**Comment by DarthDestroyer** [comment]
Not on mobile :-D

But, i think im correct

**Comment by ProtostarMusic** [comment]
I think i understand how to change the lines in the code to correct the bug, but am i opening the executable up in visual studio? or is there a DLL i need to find

**Comment by DarthDestroyer** [comment]
The fast and easiest way is:

Click on Run, wait for the compilation and look if it works. 
After that, go to the folder of the Sourcecode, there should be a new Folder... 

In that Folder should be a "OpenhardwaremonitorLib.dll" which you can copy and paste into your downloaded Folder and overwrite the "incomplete old" Lib ;-)

You could aswell use OHM from the Sourcecode Folder, too... It depends on you!

**Comment by ProtostarMusic** [comment]
I seem to get an error of 'the version of the openhardwaremonitorlib.dll is incompatible. Im running the latest alpha exe too

**Comment by ProtostarMusic** [comment]
Got it! I compiled the EXE and replaced everything there and its all working now, shows clock speeds temps and loads, thanks for the help!

**Comment by DarthDestroyer** [comment]
awesome! 

welcome in the coding world ;-)

**Comment by ProtostarMusic** [comment]
I did a year long module in the middle of my mechanical engineering degree on C++ haha! but its been several years since ive touched anything in that world so was extremely rusty haha.

**Comment by DarthDestroyer** [comment]
then, let me correct myself... welcome back ;-)
